evalml/tests/pipeline_tests/test_component_graph.py

```diff
@@ -698,7 +700,7 @@ def test_custom_input_feature_types(example_graph):
     assert input_feature_names['Imputer'] == ['column_1', 'column_2']
     assert input_feature_names['OneHot_RandomForest'] == ['column_1', 'column_2']
     assert input_feature_names['OneHot_ElasticNet'] == ['column_1', 'column_2']
-    assert input_feature_names['Random Forest'] == ['column_1_a', 'column_1_b', 'column_2_4', 'column_2_5']
+    assert input_feature_names['Random Forest'] == ['column_1_a', 'column_1_b', 'column_2_3', 'column_2_4']
```

Comment:
This change did not occur because of the reordering above; it was happening before. I think this changed because "3", "4" and "5" each have two occurrences, but the RF OHE has `top_n` set to 2, so the OHE has to choose two categories from those top three tied categories.